src/Simplex/Chat/Library/Commands.hs

```diff
@@ -575,25 +577,16 @@ processChatCommand' vr = \case
     createNoteFolderContentItems user folderId (L.map composedMessageReq cms)
   APIReportMessage gId reportedItemId reportReason reportText -> withUser $ \user ->
     withGroupLock "reportMessage" gId $ do
-      (gInfo, ms) <-
-        withFastStore $ \db -> do
-          gInfo <- getGroupInfo db vr user gId
-          (gInfo,) <$> liftIO (getGroupModerators db vr user gInfo)
-      let ms' = filter compatibleModerator ms
-          mc = MCReport reportText reportReason
+      gInfo <- withFastStore $ \db -> getGroupInfo db vr user gId
+      let mc = MCReport reportText reportReason
           cm = ComposedMessage {fileSource = Nothing, quotedItemId = Just reportedItemId, msgContent = mc, mentions = M.empty}
-      when (null ms') $ throwChatError $ CECommandError "no moderators support receiving reports"
-      let numFileInvs = length $ filter memberCurrent ms'
-      sendGroupContentMessages_ user gInfo Nothing ms' numFileInvs False Nothing [composedMessageReq cm]
-    where
-      compatibleModerator GroupMember {activeConn, memberChatVRange} =
-        maxVersion (maybe memberChatVRange peerChatVRange activeConn) >= contentReportsVersion
+      sendGroupContentMessages user gInfo (GCSMemberSupport Nothing) False Nothing [composedMessageReq cm]
```

Comment:
Does it mean that report won't be visible to admins and to members in the group itself? Not sure it is a good idea - they would be more likely to be missed, and they are higher priority than support chats.

Btw, what I don't see at all that support chats are allowed in the group - this should be in the preferences, unlike admission criteria.

Reply:
yes, and they wouldn't be missable because there's also panel in top of chat, and also they wouldn't create confusion for both members and admins that other members don't see it, and the need for this header on chat items;

there's no such preference